C01/project-metadata/C01-ProjectId612-metadata.json

```diff
@@ -69,12 +69,44 @@
       }
     },
     "type": "Feature",
+    "place_name": "Taita–Taveta, Kenya",
     "geometry": {
       "type": "Point",
       "coordinates": [
         38.802647,
         -3.910022
       ]
     }
-  }
+  },
+  "regen:previewPhoto": {
+    "@type": "schema:URL",
+    "@value": "https://regen-registry.s3.amazonaws.com/projects/kasigau/kasigau.png"
+  },
+  "regen:galleryPhotos": {
+    "@list": [
+        {
+            "@type": "schema:URL",
+            "@value": "https://regen-registry.s3.amazonaws.com/projects/kasigau/image1.png"
+        },
+        {
+            "@type": "schema:URL",
+            "@value": "https://regen-registry.s3.amazonaws.com/projects/kasigau/image2.png"
+        },
+        {
+            "@type": "schema:URL",
+            "@value": "https://regen-registry.s3.amazonaws.com/projects/kasigau/image3.png"
+        },
+        {
+            "@type": "schema:URL",
+            "@value": "https://regen-registry.s3.amazonaws.com/projects/kasigau/image4.png"
+        }
+    ]
+  },
+  "regen:landStewardStory": "In 1998, the local community supported our plan to establish the Rukinga Wildlife Sanctuary that covers 80,000 acres of forest. We established a community works project so local residents had an alternative income stream in place of poaching and clear cutting​. We brought on locally hired rangers and trained them to be wilderness guardians. We convinced the owners of the cattle to remove the cattle from the land to reduce conflict over resources.",
```

Comment:
i think this is a field we need to add to the C01 project schema..

Reply:
this is already in the generic project schema
this project is a particular case since we created it before actually defining a different project schema for C01

Reply:
@blushi so when you say the generic project schema, you mean [`ProjectPageShape`][1]? that's the one where i can find `regen:landStewardStory` but the name doesn't suggest it's a generic schema. that feels a little confusing.

[1]: https://github.com/regen-network/regen-registry-standards/blob/709a8a5382748a8dbdda761bccc1f8e4f8028d34/shacl/projects/project-page.ttl#L14

Reply:
but also, we have this metadata file labeled with this path `C01/project-metadata/C01-ProjectId612-metadata.json`. which at face value makes me think that the C01 project schema should apply. sorry if i'm getting caught up in details.

Reply:
> the generic project schema, you mean [`ProjectPageShape`](https://github.com/regen-network/regen-registry-standards/blob/709a8a5382748a8dbdda761bccc1f8e4f8028d34/shacl/projects/project-page.ttl#L14)? that's the one where i can find `regen:landStewardStory` but the name doesn't suggest it's a generic schema. that feels a little confusing.

yeah I mean the `regen:ProjectPageShape`
we had named it `ProjectPageShape`, and not just `ProjectShape` because this was primarily about data that would get shown on the project page (in some previous work, we had also been working on some project plan shape, which defines some other project related constraints, which is why we have a less generic name than just ProjectShape)

Reply:
> but also, we have this metadata file labeled with this path `C01/project-metadata/C01-ProjectId612-metadata.json`. which at face value makes me think that the C01 project schema should apply.

yeah I agree this is a bit confusing but as I said, this 612 project (kasigau) is a bit particular since we had created it prior to defining the newest C01 project shape and simplified project pages so it rather follows [ProjectPageShape](https://github.com/regen-network/regen-registry-standards/blob/709a8a5382748a8dbdda761bccc1f8e4f8028d34/shacl/projects/project-page.ttl#L14) we had at the time which expects a bit more data.

Reply:
thanks for the context @blushi this makes more sense to me now, some times i just get caught up in terms :-) going to resolve this thread now